### PR TITLE
chromium: Enable ANGLE

### PR DESCRIPTION
**Pull request by TredwellGit**
###### Motivation for this change
Notably fixes the bug that caused dragging text or images to crash the hardware renderer. Please backport.

###### Things done
Only tested and built `chromium` not `chromiumBeta` and `chromiumDev`.
- [x] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

**Comment by TredwellGit** [comment]
Unrelated to this pull request, but in the next few days I will submit another with many more Chromium fixes.

File: pkgs/applications/networking/browsers/chromium/browser.nix [hunk 1]
```diff
@@ -13,7 +13,7 @@ mkChromiumDerivation (base: rec {
 
   installPhase = ''
     mkdir -p "$libExecPath"
-    cp -v "$buildPath/"*.pak "$buildPath/"*.bin "$libExecPath/"
+    cp -v "$buildPath/"*.so "$buildPath/"*.pak "$buildPath/"*.bin "$libExecPath/"
```
Comment (primeos) [inline pkgs/applications/networking/browsers/chromium/browser.nix:16]: Could you note in a comment which shared objects this installs (would make the review easier)?

**Review by primeos — COMMENTED** [review]
Thanks, that was on my TODO list for M86 and this makes things way easier :)
Did you happen to test this with VA-API? IIRC this should cause problems on M85. But we can merge it in ~4 days anyway when M86 is released.

Looking forward to your other changes (Chromium needs more love and maintainers).

**Comment by TredwellGit** [comment]
>Did you happen to test this with VA-API?

No, I never managed to get VA-API working in the first place. 

>Could you note in a comment which shared objects this installs (would make the review easier)?

It is probably not necessary to copy all the files it that currently does, but I will fix that in a later pull request.

>Looking forward to your other changes (Chromium needs more love and maintainers).

I am going to wait until this is merged.

**Comment by TredwellGit** [comment]
https://github.com/NixOS/nixpkgs/pull/99975

**Pull request closed without merging**